Clarify intelligent timing tie-breaking behavior for SMS (#9646)
braze-inc/braze-docs: _docs/_user_guide/brazeai/intelligence/intelligent_channel.md

To enable the Intelligent Channel filter, select the **Intelligent Channel** filter on the **Target Users** page when creating a email, web push, or mobile push campaign.

{% alert important %}

> To compute the engagement rate of the SMS channel, turn on [SMS link shortening]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/link_shortening/#overview/) with advanced tracking and click tracking.  Without this tracking, SMS may still be selected as the Intelligent Channel for a 0% engagement rate due to the [Tie-breaking behavior]({{site.baseurl}}/user_guide/brazeai/intelligence/intelligent_channel/#tie-breaking).

Reviewer: ```suggestion
To compute the engagement rate of the SMS channel, turn on [SMS link shortening]({{site.baseurl}}/user_guide/message_building_by_channel/sms/campaign/link_shortening/#overview/) with advanced tracking and click tracking. Without this tracking, SMS may be selected as the Intelligent Channel for a 0% engagement rate because of our [tie-breaking behavior]({{site.baseurl}}/user_guide/brazeai/intelligence/intelligent_channel/#tie-breaking).
```
